packages/cli/src/util/dev/server.ts

```diff
@@ -703,6 +703,10 @@ export default class DevServer {
     buildEnv['NOW_REGION'] = 'dev1';
     allEnv['NOW_REGION'] = 'dev1';
 
+    // simulate parts of the platform for local environment
+    runEnv['VERCEL_ENV'] = 'development';
+    runEnv['VERCEL'] = '1';
```

Comment:
These should apply at "build time" as well:

```suggestion
    allEnv['VERCEL_ENV'] = 'development';
    allEnv['VERCEL'] = '1';
```